src/udiskslinuxdriveata.c

```diff
@@ -470,6 +470,36 @@ static gboolean get_pm_state (UDisksLinuxDevice *device, GError **error, guchar
   return rc;
 }
 
+/**
+ * udisks_linux_drive_ata_is_active:
+ * @device: The ata device to get pm state from
+ * @error: Return location for error or %NULL
+ *
+ * Get the current power mode status from a drive.
+ *
+ * The format of @state is the result obtained from sending the
```

Comment:
So `state` is not propagated outside anymore, please remove the related doc strings.